packages/navstar/extra/dcos-navstar.service

```diff
@@ -5,7 +5,7 @@ Description=Navstar: A distributed systems & network overlay orchestration engin
 Restart=always
 StartLimitInterval=0
 RestartSec=5
-WorkingDirectory=${PKG_PATH}/navstar
+WorkingDirectory=/opt/mesosphere/active/navstar/navstar
```

Comment:
Don't think we want to do this. The build script should be taking care of replacing PKG_PATH here. There are other examples in the repo.